README.md

```diff
@@ -53,6 +53,16 @@ Contributions are welcome, as is feedback.
 * [SwiftOCR](https://github.com/garnele007/SwiftOCR) - fast and simple OCR library written in Swift
 * [Attention-OCR](https://github.com/da03/Attention-OCR) - OCR engine using visual attention mechanisms
 
+Older (abandoned?) OCR engines
```

Comment:
This should be a headline, i.e. you want to add `###` in the beginning.

Maybe, we should simply write `Older and possibly abandoned OCR engines`.